kafka-connect-s3/docs/s3_connector.rst

```diff
@@ -122,6 +122,8 @@ Every service will start in order, printing a message with its status:
 .. note:: You need to make sure the connector user has write access to the S3 bucket
    specified in ``s3.bucket.name`` and has deployed credentials
    `appropriately <http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html>`_.
+   You can also pass additional properties to credential provider, please refer to the
+   `Configurable credential provider`_ section.
```

Comment:
`credentials` here and below in other places.

Reply:
thanks, fixed